applications/tari_validator_node/proto/dan/network.proto

```diff
@@ -46,3 +46,24 @@ message GetPeersResponse {
   repeated bytes addresses = 2;
   IdentitySignature identity_signature = 3;
 }
+
+message VNStateSyncRequest {
+  // Range
+  tari.dan.common.ShardId start_shard_id = 1;
```

Comment:
These are probably not needed

Reply:
Hey @stringhandler, I removed these from use. Let me know if you think it is fine !